### PR TITLE
Prompt for test suite from CLI

### LINKED ISSUE #2339: CLI - scope unit tests to run
In order of importance:
- [x] Run All Test Suites for a Document
- [ ] Run everything <- Not doing this because user _must_ provide an environment to use from CLI.
- [x] Run Individual Tests from a Suite

### PR DESCRIPTION
**Pull request by develohpanda**
image

### Identify suite, scope by test name, specify environment:
image

### Identify document, scope by test name, and keep generated test file
image

### Identify document, scope by suite name
image

### Run all suites in a document, bail on first error
image

### Disable prompts with `--ci` global option

TODO:
- [x] Unit tests
- [x] Improve scoping options tests (eg. entire ApiSpec (workspace), test suite, specific test)
- [x] Responsiveness of CLI... it's not snappy, possibly because of the babel to webpack switch
- [x] Seems like tests are returning a fail exit code, even if they pass
- [x] Prompt for environment
- [x] Add `--ci` to disable prompts
- [x] Support `--keep-file` option
- [x] Support `--env` option
- [x] Handle identical suite names
- [x] Handle identical document names
- [x] `insomnia.send()` does not work from CLI because the `activeRequestId` is not set correctly
- [ ] `mustFindSingleOrNone` and `mustFindSingle` should fail with a more friendly message (currently prints exception in console)
  image

Closes #2339

### CODE REVIEW

File: packages/insomnia-inso/src/commands/run-tests.js [hunk 1]
```diff
@@ -29,40 +31,45 @@ function validateOptions({ reporter }: RunTestsOptions): boolean {
   return true;
 }
 
-export async function runInsomniaTests(options: RunTestsOptions): Promise<boolean> {
+const createTestSuite = (dbSuite: UnitTestSuite, dbTests: Array<UnitTest>) => ({
+  name: dbSuite.name,
+  tests: dbTests.map(({ name, code, requestId }) => ({ name, code, requestId })),
+});
```
Comment (gschier) [inline packages/insomnia-inso/src/commands/run-tests.js]:
```suggestion
const createTestSuite = (dbSuite: UnitTestSuite, dbTests: Array<UnitTest>) => ({
  name: dbSuite.name,
  tests: dbTests.map(({ name, code, requestId }) => ({ name, code, defaultRequestId: requestId })),
});
```

File: packages/insomnia-inso/__mocks__/insomnia-testing.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
-const mod = jest.requireActual('insomnia-testing');
+// const mod = jest.requireActual('insomnia-testing');
```
Comment (gschier) [inline packages/insomnia-inso/__mocks__/insomnia-testing.js:1]: This supposed to be commented out?

File: packages/insomnia-inso/src/logger.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+export const noConsoleLog = async <T>(callback: () => Promise<T>): Promise<T> => {
+  const oldConsoleLog = console.log;
+  (console: Object).log = () => {};
+
+  try {
+    return await callback();
+  } finally {
+    (console: Object).log = oldConsoleLog;
+  }
+};
```
Comment (gschier) [inline packages/insomnia-inso/src/logger.js:11]: Nice, this is awesome!

**Review by gschier — APPROVED** [review]
Nice work @develohpanda!

File: packages/insomnia-inso/__mocks__/insomnia-testing.js (hunk 2 above)
Comment (develohpanda) [reply packages/insomnia-inso/__mocks__/insomnia-testing.js:1]: Yeah, I should delete the commented line

**Comment by gleb-grigoriev** [comment]
Hi.
Is it possible to save test report in junit format as a file?

**Pull request merged**